app/components/ui/PeoplePicker/PeoplePickerLogic.test.js

```diff
@@ -71,8 +77,8 @@ describe('PeoplePicker', () => {
     expect(render).toHaveBeenCalled()
   })
 
-  it('renders as many person pods as people', () => {
-    const wrapper = makeWrapper()
+  it('renders as many person pods as people', async () => {
+    const wrapper = await makeWrapper()
```

Comment:
Looking back through this I'm unsure we need this function to be async and await the makeWrapper either.

Reply:
👍